tutorials/Getting_Started/README.md

```diff
@@ -1,63 +1,61 @@
-# Getting Started with Compose for Desktop
+# Getting Started With Compose For Desktop
 
-## What is covered
+## What Is Covered
 
 In this tutorial we will create a simple desktop UI application
 using the Compose UI framework.
 
-## Prerequisites
+### Prerequisites
 
- Compose for Desktop can produce applications for macOS, Linux and Windows platforms,
-and so any of these platforms can be used for this tutorial.
+ Compose for Desktop can produce applications for macOS, Linux, and Windows platforms. So any of these platforms can be used for this tutorial.
 
 The following software has to be preinstalled:
    * JDK 11 or later
    * IntelliJ IDEA Community Edition or Ultimate Edition 20.2 or later (other editors could be used, but we assume you are using IntelliJ IDEA in this tutorial)
 
-## Creating a new project
+## Creating A New Project
 
-### New project wizard
+### New Project Wizard
 
-Kotlin support in IDEA IDE starting with the version 2020.3 comes with the new project wizard
-capable to create a Compose application automatically.
+Kotlin support in IDEA IDE starting with the version 2020.3, comes with the new project wizard which is
```

Comment:
IntelliJ IDEA version 2020.3 comes with the new project wizard which is capable of creating a Compose application automatically.

Reply:
@B1ggDave is there a way that I can click a button or something to use your comments? Or do I have to make the Changes on my local system and make another commit? Sorry I am still new at this.

Reply:
If I'm not wrong B1ggDave just commented without suggesting changes (where you can simply accept the correction by a button click). So you probably have to make the changes manually and make another commit.

_Unless I can't see this information because I'm not the author of the PR. Have a look at the [Files changed](https://github.com/JetBrains/compose-jb/pull/534/files) tab, you should find it there._